index2.html

```diff
@@ -22,7 +22,7 @@
   <script src="node_modules/webrtc-adapter/out/adapter.js"></script>
   <script src="node_modules/getusermedia-js/dist/getUserMedia.min.js"></script>
 	
-  <script src="dist/infragram2.js"></script>
+  <script src="dist/infragram.js"></script>
```

Comment:
Very cool! So do we need to keep dist/infragram2.js now? Just curious, thanks for all the thorough testing! I'd love to hear from @stephaniequintana on this as well, just a review would be helpful! Thank you!

Reply:
I think there's no need to keep the dist/infragram2.js file.In the mean time please permit me move forward with the merge.

Reply:
@stephaniequintana, your review is very important on this topic. Please I will consider and apply it on the next PR. For now please  let me merge this.